{
  "id": "CVE-2020-13328",
  "published": "2020-09-30T18:15:19",
  "modified": "2024-05-30T03:01:23.398367",
  "details": "An issue has been discovered in GitLab affecting versions prior to 13.1.2, 13.0.8 and 12.10.13. GitLab was vulnerable to a stored XSS by using the PyPi files API.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:H/UI:R/S:C/C:L/I:L/A:N"
    }
  ],
  "repo_url": "https://gitlab.com/gitlab-org/gitlab",
  "cwes": [],
  "commits": [
    "0cf2ca9deb2facdaa52e706ce4e1cadc484516d9"
  ]
}
